src/js/ble_uuid_util.js

```diff
@@ -36,8 +36,23 @@
 
 module.exports.removeDashes = function(uuid) {
   if (uuid) {
-    uuid = uuid.replace(/-/g, '');
+    uuid = uuid.split('-').join('');
   }
 
   return uuid;
 };
+
+module.exports.tokenize = function(str) {
+  str = String(str);
+  var len = str.length;
+  var arr = [];
+  var i = 0;
+
+  while (i < len - (len % 2)) {
+    arr.push(str.charAt(i) + str.charAt(i + 1));
+    i = i + 2;
+  }
+
+  if (i < len) arr.push(str.charAt(i));
+  return arr;
+}
```

Comment:
It seems the code always uses .reverse().join('') after this call, perhaps we could do that here.